translations/messages.nl_NL.xliff

```diff
@@ -112,6 +112,11 @@
         <target>Token gevet</target>
         <jms:reference-file line="42">/src/../src/Surfnet/StepupRa/RaBundle/Resources/views/translations.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="ad15ff92077a3cd0f1cb0f796374d091bbf76cac" resname="ra.auditlog.action.vetted_possession_unknown">
+        <source>ra.auditlog.action.vetted_possession_unknown</source>
+        <target>Token vetted</target>
```

Comment:
Maybe you could translate this best effort using the available translations.

Reply:
I've consulted the PO and it was OK for now to leave it as-is. For an RA it wouldn't be very useful.